File: app/src/main/java/com/example/tribecrypto/java/MainActivity.java
package com.example.tribecrypto.java;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import androidx.viewpager.widget.ViewPager;

import com.example.tribecrypto.R;
import com.example.tribecrypto.adapter.ViewPagerAdapter;
import com.google.android.material.tabs.TabLayout;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ViewPagerAdapter adapter = new ViewPagerAdapter(getSupportFragmentManager());
        ViewPager viewPager = findViewById(R.id.viewViewPager);
        TabLayout viewTabLayout = findViewById(R.id.viewTabLayout);
        viewPager.setAdapter(adapter);
        viewTabLayout.setupWithViewPager(viewPager, true);
    }
}

File: app/src/main/java/com/example/tribecrypto/MainActivity.kt
package com.example.tribecrypto

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import com.example.tribecrypto.adapter.ViewPagerAdapter
import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.toolbar_layout.view.*

class MainActivity : AppCompatActivity() {

//    private lateinit var toolbar: Toolbar

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

//        toolbar = viewToolbar as Toolbar
//        setSupportActionBar(toolbar)

        val viewPagerAdapter = ViewPagerAdapter(supportFragmentManager)
        viewViewPager.adapter = viewPagerAdapter
        viewTabLayout.setupWithViewPager(viewViewPager, true)
    }
}
